### ISSUE TITLE
The Due Date hotkey 'd' doesn't open the due date option

### ISSUE THREAD

**Issue by ncallaway** [go-vikunja/vikunja#3818]
### Pre-submission checklist

- [x] I have searched for existing open or closed issue reports with the same problem.

### Description

**Reproduction Steps**

1. Visit the detail page for a task that does not have a due date
2. Press 'd'
3. The due date field appears in the main area, and flashes, but does not open
    - To open the due date with the keyboard, you now have to tab over to put a bunch.
4. Visit the detail page for a task that has a due date
5. Press 'd'
6. Nothing happens

**Expected Behavior**

- At step 3, the due date field should appear *and* open, so it's keyboard navigable
- At step 6, they due date field should open so it's keyboard navigable.

### Vikunja Version

2.6.0

### Browser and version

Firefox 153.0

### Can you reproduce the bug on the Vikunja demo site?

Yes
